deployments/gpu-operator/values.yaml

```
@@ -358,9 +358,42 @@ migManager:
     - name: WITH_REBOOT
       value: "false"
   resources: {}
+  # MIG configuration
+  # Use "name" to either point to an existing ConfigMap or to create a new one with a list of configurations(i.e with create=true).
+  # Use "data" to build an integrated ConfigMap from a set of configurations as
+  # part of this helm chart. An example of setting "data" might be:
+  # config:
+  #   name: custom-mig-parted-configs
+  #   create: true
+  #   data: |-
+  #     all-disabled:
+  #       - devices: all
+  #         mig-enabled: false
+  #     custom-mig:
+  #       - devices: [0]
+  #         mig-enabled: false
+  #       - devices: [1]
+  #          mig-enabled: true
+  #          mig-devices:
+  #            "1g.10gb": 7
+  #       - devices: [2]
+  #         mig-enabled: true
+  #         mig-devices:
+  #           "2g.20gb": 2
+  #           "3g.40gb": 1
+  #       - devices: [3]
+  #         mig-enabled: true
+  #         mig-devices:
+  #           "3g.40gb": 1
+  #           "4g.40gb": 1
   config:
-    name: "default-mig-parted-config"
     default: "all-disabled"
+    # Create a ConfigMap (default: false)
+    create: false
+    # ConfigMap name (either exiting or to create a new one with create=true above)
+    name: "default-mig-parted-config"
```

Let's make the default name empty to align with the values for the device-plugin. 
```suggestion
    name: ""
```

---

@cdesiniotis I am unsure if we should do this, as we deploy this configmap by default when the mig-manager is enabled.

---

Ah ok, I do see [here](https://github.com/NVIDIA/gpu-operator/blob/7068fec931e78af2994f82a6ecfea55cbf737b48/controllers/object_controls.go#L1744) that we inject `default-mig-parted-config` when it's unspecified

Please disregard my comment

---

@frittentheke I do not see this comment being addressed. Can you update this as I have suggested? The `name` field here should only be used when configuring a custom ConfigMap, so defaulting to `default-mig-parted-config` does not make sense.